### PR TITLE
fix(driver): update param count of seccomp enter event

### PR DESCRIPTION
**Pull request by jasondellaluce**
**What type of PR is this?**

/kind bug

**Any specific area of the project related to this PR?**

/area driver-kmod

/area driver-bpf

/area driver-modern-bpf

**Does this PR require a change in the driver versions?**

/version driver-SCHEMA-version-minor

**What this PR does / why we need it**:

Ports the changes of https://github.com/draios/sysdig/pull/1505.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by Andreagit97 — DISMISSED** [review]
Thanks!
/approve

**Comment by Andreagit97** [comment]
Opss this is not enough, you have to change the code in the modern probe, see the CI failed test
```
; PUSH_FIXED_SIZE_TO_RINGBUF(ringbuf, param, sizeof(u64));
118: (bf) r3 = r2
119: (77) r3 >>= 56
120: (73) *(u8 *)(r1 +7) = r3
 R0=inv(id=0) R1_w=mem(id=0,ref_obj_id=4,off=30,imm=0) R2_w=inv(id=8) R3_w=inv(id=0,umax_value=255,var_off=(0x0; 0xff)) R6=ptr_pt_regs(id=0,off=272,imm=0) R7=mem(id=0,ref_obj_id=4,off=0,imm=0) R8=invP2 R9=inv8 R10=fp0 fp-8=mmmmmmmm fp-16=inv2 refs=4
invalid access to memory, mem_size=36 off=37 size=1
R1 min value is outside of the allowed memory range
processed 119 insns (limit 1000000) max_states_per_insn 0 total_states 6 peak_states 6 mark_read 4
-- END PROG LOAD LOG --
libbpf: prog 'seccomp_e': failed to load: -13
libbpf: failed to load object 'bpf_probe'
libbpf: failed to load BPF skeleton 'bpf_probe': -13
libpman: failed to load BPF object (errno: 13 | message: Permission denied)
Unable to open the engine: 
```

File: driver/event_table.c [hunk 1]
```diff
@@ -347,7 +347,7 @@ const struct ppm_event_info g_event_info[] = {
 	[PPME_SYSCALL_SETPGID_X] = {"setpgid", EC_PROCESS | EC_SYSCALL, EF_MODIFIES_STATE, 1, {{"res", PT_PID, PF_DEC} } },
 	[PPME_SYSCALL_BPF_E] = {"bpf", EC_OTHER | EC_SYSCALL, EF_CREATES_FD | EF_OLD_VERSION, 1, {{"cmd", PT_INT64, PF_DEC} } },
 	[PPME_SYSCALL_BPF_X] = {"bpf", EC_OTHER | EC_SYSCALL, EF_CREATES_FD | EF_OLD_VERSION, 1, {{"res_or_fd", PT_DYN, PF_DEC, bpf_dynamic_param, PPM_BPF_IDX_MAX} } },
-	[PPME_SYSCALL_SECCOMP_E] = {"seccomp", EC_OTHER | EC_SYSCALL, EF_NONE, 1, {{"op", PT_UINT64, PF_DEC}, {"flags", PT_UINT64, PF_HEX} } },
```
Comment (FedeDP) [inline driver/event_table.c:350]: :face_exhaling:

**Review by FedeDP — APPROVED** [review]
/approve

**Comment by jasondellaluce** [comment]
Thanks for adjusting it Andre!

**Review by Andreagit97 — APPROVED** [review]
/approve

**Pull request merged**